blue_hardware_interface/src/blue_kinematics.cpp

```diff
@@ -220,6 +220,8 @@ void BlueKinematics::setJointOffsets(
   std::lock_guard<std::mutex> lock(kinematics_mutex_);
 
   joint_offsets_ = offsets;
+  // set the gripper offset to be a realtive offset to allow for gripper calibration
```

Comment:
*relative
*Set

also hacky -- should we fix before merging?

Reply:
we can fix this with the gripper only calibration service

Reply:
sure; i can do this when you're done with your changes